### DISCUSSION TITLE
How to interpret previously failed files?

### DISCUSSION THREAD

**Discussion by abdcef** [brianpetro/obsidian-smart-connections#136] [Q&A]
![image](https://user-images.githubusercontent.com/56148250/228760492-954bed83-8dfb-45b2-ad1c-2551445f3cc3.png)

Does this mean that the embeddings do not contain the content where the files failed? Do I need to check this occasionally and retry failed files manually?

**Comment by brianpetro** [comment]
Yes, if you're being notified, then try running the retry failed files button. 

Not sure what that blank line is about. You could deleted the "failed-embeddings" file to see if that solves it. It will be recreated if any files fail in the future.

**Reply by wdmason** [reply]
I had a similar issue at first as well. 

It was solved after I had successfully registered a valid ChatGPT API. Once that was done, I manually retried, and all of the files were successfully processed.

**Reply by dougkeiller** [reply]
I have a long list of Previously Failed Files. I've retried multiple times.

Is there any documentation showing what causes failed files? Characters in file names, etc?

All of mine are related to the Omnivore plugin's directory and automatically downloaded articles for reading

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/128188043/0dc8ba5e-601b-4b38-9590-3ce2dc22fa38)

**Reply by brianpetro** [reply]
@dougkeiller it looks likely to be some sort of issue with the filename. Unfortunately there isn't an exact parity between the file names that Obsidian can handle and those that Smart Connections can handle. 

If you check the developer console for error logs when you retry embedding the failed files, then that might give us some insights into exactly what's going wrong. If you do that, you should see some red error logs that you can screenshot and then I can give them a look. 

🌴

**Reply by dougkeiller** [reply]
Thanks, Brian

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/128188043/06dc889e-4f37-45c5-8fa8-6277d31069c6)
